### PR TITLE
#179: made more models work

### PR DESCRIPTION
**Pull request by schuelet**
most models run, though some are very slow (2min+) even on the default simulations
Package gsl does not work
Package r2OpenBugs can't find the openbugs program on the computer
IMPORTANT NOTE: PiresOutblA2011-Salmonells.fskx had a faulty Salmonella-all.csv file resource which I fixed manually (some entries had a newline character, breaking the file). R seems to ignore it, but Renjin (and KNIME CSV Reader) cant read the file properly).

### CODE REVIEW

File: src/Server.kt [hunk 1]
```diff
@@ -28,7 +28,7 @@ import org.renjin.script.RenjinScriptEngine
 import org.renjin.script.RenjinScriptEngineFactory
 
 val MAPPER = ObjectMapper()
-const val LOCAL_TEST = false // Set true for testing locally with netty.
+const val LOCAL_TEST = true // Set true for testing locally with netty.
```
Comment (miguelalba) [inline src/Server.kt:31]: We need to leave it false for it to run when deployed on Tomcat.

File: src/Server.kt [hunk 2]
```diff
@@ -168,12 +168,21 @@ fun Application.module() {
 
         get("/execute/{i}") {
             // TODO: execute model at index i
-            val testModelPath = {}.javaClass.getResource("/static/files/preparedQuick.fskx").file
-            val testModelFile = File(testModelPath)
-            val model = readModel(testModelFile)
-            val svgPlot = model.run()
+            //val testModelPath = {}.javaClass.getResource("/static/files/preparedQuick.fskx").file
```
Comment (miguelalba) [inline src/Server.kt:171]: Please remove commented code.

File: src/Server.kt [hunk 3]
```diff
@@ -280,7 +289,7 @@ data class FskModel(
     val modelScript: String,
     val visualizationScript: String?,
     val simulations: List<FskSimulation>,
-    val selectedSimulationIndex: Int,
+    var selectedSimulationIndex: Int,
```
Comment (miguelalba) [inline src/Server.kt:292]: Is selectedSimulationIndex necessary? We are taking the index from simulations in runSelectedSimulation.

**Review by miguelalba — CHANGES_REQUESTED** [review]
Some lesser problems are commented. The order of the parameters is lost in SED-ML document and generated simulation scripts for the model *Fitting Distribution for Microbial Counts* but I can restore the parameters order from the metadata where it's preserved.

File: src/Server.kt (hunk 3 above)
Comment (schuelet) [reply src/Server.kt:291]: I dont know, this was already in master. We can change it once the simulation execution takes simulations.

**Review by miguelalba — COMMENTED** [review]
I am adding my commit with changes to the simulation.

**Comment by miguelalba** [comment]
This PR was replaced with https://github.com/RakipInitiative/landingpage/pull/8. @schuelet can we delete the branch?

**Pull request closed without merging**